src/components/logic/operate.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable radix */
+/* eslint-disable import/extensions */
+import { Big } from 'big.js';
+
+const Operate = (nOne, nTwo, operation) => {
+  const None = Big(nOne);
+  const Ntwo = Big(nTwo);
+  let Total = '';
+  if (operation === '+') {
+    Total = None + Ntwo;
+  }
+  if (operation === '-') {
+    Total = None - Ntwo;
+  }
+  if (operation === '/') {
+    Total = None / Ntwo;
+  }
+  if (operation === '*') {
+    Total = None * Ntwo;
+  }
+  if (operation === '%') {
+    Total = None % Ntwo;
+  }
```

Comment:
When doing arithmetic calculation inside `operate` function you need to use the Big.js api and NOT the standard javascript operators`(+,-./.%,*)`
Checkout [Big.js docs](https://mikemcl.github.io/big.js/#) on how to use the library.
`hint:`
```javascript
// addition
const x = new Big(0.1)
const y = x.plus(0.2)            // '0.3'
Big(0.7).plus(x).plus(y)   // '1.1'
```

Reply:
Ok, i got it,  and fixed !!, thanks for the notice